pkgs/kde/gear/gwenview/default.nix

```diff
@@ -2,6 +2,7 @@
   mkKdeDerivation,
   qtsvg,
   qtwayland,
+  qtimageformats,
```

Comment:
Can you add a comment for this? It's the only non-hard dependency in the list, and it would be nice to know why it was added later.